Index: JSPWiki-2.4.71/src/com/ecyrd/jspwiki/parser/JSPWikiMarkupParser.java
===================================================================
--- JSPWiki-2.4.71/src/com/ecyrd/jspwiki/parser/JSPWikiMarkupParser.java	(revision 25385)
+++ JSPWiki-2.4.71/src/com/ecyrd/jspwiki/parser/JSPWikiMarkupParser.java	(working copy)
@@ -413,8 +413,12 @@
 
             case EDIT:
                 el = new Element("a").setAttribute("class",CLASS_EDITPAGE);
-                el.setAttribute("title","Create '"+link+"'");
-                el.setAttribute("href", m_context.getURL(WikiContext.EDIT,link));
+                //el.setAttribute("title","Create '"+link+"'");
+                String plainLink = m_context.getURL(WikiContext.EDIT,link);
+                if(plainLink.startsWith("Edit.jsp?page=")) {
+                    plainLink = plainLink.substring(14);
+                }
+                el.setAttribute("href", plainLink);
                 el.addContent(text);
                 break;
 
@@ -1367,7 +1371,7 @@
                 }
                 else
                 {
-                    makeLink( ATTACHMENT, attachment, link, null );
+                    makeLink( EDIT, reallink, link, null );
                 }
             }
             else if( (hashMark = reallink.indexOf('#')) != -1 )
@@ -2297,7 +2301,7 @@
                 {
                     // DOS line feeds we ignore.
                 }
-                else if( ch == '<' )
+                /*else if( ch == '<' )
                 {
                     m_plainTextBuf.append( "&lt;" );
                 }
@@ -2308,7 +2312,7 @@
                 else if( ch == '&' )
                 {
                     m_plainTextBuf.append( "&amp;" );
-                }
+                }*/
                 else if( ch == '~' )
                 {
                     String braces = readWhile("}");
