Index: JSPWiki-2.4.71/src/com/ecyrd/jspwiki/parser/MarkupParser.java
===================================================================
--- JSPWiki-2.4.71/src/com/ecyrd/jspwiki/parser/MarkupParser.java	(Revision 20728)
+++ JSPWiki-2.4.71/src/com/ecyrd/jspwiki/parser/MarkupParser.java	(Arbeitskopie)
@@ -67,7 +67,7 @@
     /** Lists all punctuation characters allowed in WikiMarkup. These
         will not be cleaned away. */
     
-    protected static final String           PUNCTUATION_CHARS_ALLOWED = "._";
+    protected static final String           PUNCTUATION_CHARS_ALLOWED = "._-";//added a dash so links like hello-world.html would be shown as expected
 
     protected MarkupParser( WikiContext context, Reader in )
     {
@@ -252,7 +252,7 @@
             {
                 // Is a letter
                 
-                if( isWord ) ch = Character.toUpperCase( ch );
+                if( isWord ) ch = ch;//Character.toUpperCase( ch );
                 clean.append( ch );
                 isWord = false;
             }
